src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { funFacts,  FunFact  as FunFactInterface } from "./funfacts";
+import "./FunFactPage.scss";
+
+interface FunFactProps {
+    funFacts: FunFactInterface[];
```

Comment:
Too large an indent:
```suggestion
  funFacts: FunFactInterface[];
```